psycop/common/model_training_v2/hyperparameter_suggester/optuna_hyperparameter_search.py

```diff
@@ -0,0 +1,108 @@
+import copy
+from pathlib import Path
+from typing import Any, Literal
+
+import optuna
+from confection import Config
+from optuna import Study, Trial
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.config.baseline_schema import BaselineSchema
+from psycop.common.model_training_v2.hyperparameter_suggester.hyperparameter_suggester import (
+    SuggesterSpace,
+    suggest_hyperparams_from_cfg,
+)
+from psycop.common.model_training_v2.hyperparameter_suggester.suggesters.base_suggester import (
+    Suggester,
+)
+
+
+class OptunaHyperParameterOptimization:
+    @staticmethod
+    def _validate_configspace(cfg: dict[str, Any]) -> None:
```

Comment:
Perhaps rename to make clear that the only validation it does is to check that it has suggesters?